great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -653,32 +653,30 @@ def expect_column_median_to_be_between(self,
                 sa.func.sum(
                     sa.case([(sa.column(column) == None, 1)], else_=0)
                 ).label('null_count')
-            ]).
-            select_from(sa.table(self.table_name))
+            ]).select_from(sa.table(self.table_name))
```

Comment:
Did you merge the branch that cleans up warnings into `feature/setup_entrypoints`?

Reply:
Looks like not. I thought it would see that those commits are already merged, but I'll rebase off current develop...